<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCommandtest</class>
 <widget class="QWidget" name="QtCommandtest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCommandtest</string>
  </property>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>测试界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>指令列表</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>871</width>
     <height>691</height>
    </rect>
   </property>
   <property name="title">
    <string>指令组成</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>指令执行选择</string>
    </property>
    <widget class="QComboBox" name="comboBox_exeattr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>立即执行</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>延时执行</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>指令执行属性</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>延迟执行时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>指令有效数据</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_reminder">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>440</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>指令生成提示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已选指令列表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>组帧完指令列表</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_selectcmd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buildup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_validdata">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_build_frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>171</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>40</y>
     <width>201</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>解析结果</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_cmd_send">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>760</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>执行遥控指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clr_cmd">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>760</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清除遥控指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_selectlist">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_del_cmd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>151</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_send_Identification">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>800</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>身份识别</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_bind_station">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>840</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定地面站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_server">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>840</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TCP到运控</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_reply_telemetry">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>480</y>
     <width>191</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>810</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>测试遥控</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_to_udp">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>840</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接udp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>840</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>tcp连接到江哥</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_test</sender>
   <signal>clicked()</signal>
   <receiver>QtCommandtest</receiver>
   <slot>btn_to_gather()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCommandtest</receiver>
   <slot>test_btn_to_connect_tcp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>846</x>
     <y>767</y>
    </hint>
    <hint type="destinationlabel">
     <x>811</x>
     <y>782</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtCommandtest</receiver>
   <slot>tcp_connect_ground_check_as_client()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1020</x>
     <y>850</y>
    </hint>
    <hint type="destinationlabel">
     <x>1018</x>
     <y>855</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_first_clicked()</slot>
  <slot>btn_clr_cmd()</slot>
  <slot>btn_to_connect_tcp()</slot>
  <slot>btn_to_gather()</slot>
  <slot>tcp_connect_ctl_center_as_client()</slot>
  <slot>tcp_send_connect_data_to_ctl_center()</slot>
  <slot>tcp_send_identity_data_to_ctl_center()</slot>
  <slot>test_btn_to_connect_tcp()</slot>
  <slot>tcp_connect_ground_check_as_client()</slot>
 </slots>
</ui>
